examples/rendering_plane_widget.py

```diff
@@ -0,0 +1,38 @@
+import napari
+from skimage import data
+
+from napari_threedee.manipulators import RenderPlaneManipulator
+from napari_threedee.manipulators.qt_render_plane_manipulator import QtRenderPlaneManipulatorWidget
+
+viewer = napari.Viewer(ndisplay=3)
+
+# add a volume
+blobs = data.binary_blobs(
+    length=64, volume_fraction=0.1, n_dim=3
+).astype(float)
+volume_layer = viewer.add_image(
+    blobs, rendering='mip', name='volume', blending='additive', opacity=0.25
+)
+
+# add the same volume and render as plane
+# plane should be in 'additive' blending mode or depth looks all wrong
+plane_parameters = {
+    'position': (32, 32, 32),
+    'normal': (1, 0, 0),
+    'thickness': 10,
+    'enabled': True
+}
+
+plane_layer = viewer.add_image(
+    blobs,
+    rendering='average',
+    name='plane',
+    colormap='bop orange',
+    blending='additive',
+    opacity=0.5,
+    experimental_slicing_plane=plane_parameters
+)
+
+widget = QtRenderPlaneManipulatorWidget(viewer)
+viewer.window.add_dock_widget(widget)
+napari.run()
```

Comment:
this doesn't work yet because `layer=None` isn't well handled in the `Manipulator` init 

To make this work I think we need to either recreate the vispy mesh node or change its parent in the manipulator

Reply:
Good point! I think we can delay creation of the mesh until the layer is not `None` (for example in the `set_layer` method. I think it would be okay to make an issue and then make a follow up PR.